website/client/components/groups/membersModal.vue

```diff
@@ -249,24 +325,34 @@ export default {
 
       if (this.searchTerm) {
         sortedMembers = sortedMembers.filter(member => {
-          return member.profile.name.toLowerCase().indexOf(this.searchTerm.toLowerCase()) !== -1;
+          return (
+            member.profile.name
+              .toLowerCase()
+              .indexOf(this.searchTerm.toLowerCase()) !== -1
+          );
         });
       }
 
-      if (!this.sortOption) return sortedMembers;
-
-      sortedMembers = sortBy(this.members, [(member) => {
-        if (this.sortOption === 'tier') {
-          if (!member.contributor) return;
-          return member.contributor.level;
-        } else if (this.sortOption === 'name') {
-          return member.profile.name;
-        } else if (this.sortOption === 'lvl') {
-          return member.stats.lvl;
-        } else if (this.sortOption === 'class') {
-          return member.stats.class;
-        }
-      }]);
+      // Check if sortOption is an empty object
+      if (isEmpty(this.sortOption)) return sortedMembers;
+
+      // sortedMembers = sortBy(this.members, [
```

Comment:
is this code commented because it might be useful in future? If not let's remove this

Reply:
my PC has throwing a fit when running the unit tests, so I kept them in as backup. Since all the CI tests have passed I will remove them.